src/components/Header.jsx

```diff
@@ -0,0 +1,37 @@
+import { Heading, Center, Box } from "@chakra-ui/react";
+
+export function Header() {
+  return (
+    <>
+      <Box
+        bg="#e49800"
+        fontFamily="Londrina Solid, cursive"
+        textAlign="center"
+        padding="5rem 0 2.5rem 0"
+      >
```

Comment:
The same as in Footer. 
1. Whole header should be in "header" tag
2. You can use chakra spacing system for padding
3. Better way for using different font family is to define it in theme (look at https://chakra-ui.com/docs/styled-system/theming/theme#typography) or at least define constant in separated file to share fontFamily names